docs/requirements/gen3-tracker.md

```diff
@@ -54,11 +54,11 @@ username: someone@example.com
 
 The following options and environmental variables are synonymous, you may set them as environmental variables or pass them as parameters to the command line.
```

Comment:
"**make sure to** set them" as something that needs to be done. Also, I think we should mention that we carry on with the rest of the quickstart as if we have chosen to set environmental variables